package.json

```diff
@@ -74,6 +74,7 @@
   "scripts": {
     "browserstack": "grunt build && sh build/run-browserstack.sh",
     "build": "grunt build",
+    "dev": "grunt watch",
```

Comment:
Yes, I can put in a separate commit if wanted, but I'm including it because I mention using `yarn dev` to start a local development work flow. I find having the npm/yarn script to be a better approach than a specific grunt task, because we can change them later if needed.